frontend/storybook/src/components/entry/user/login/pages/landing.ts

```diff
@@ -2,15 +2,15 @@ import {argsToAttrs} from "@utils/attributes";
 import "@elements/entry/user/login/pages/landing";
 import "@elements/core/buttons/rectangle";
 import "@elements/entry/user/_common/buttons/google";
-import "@elements/entry/user/login/footer/register";
 
 export default {
   title: 'Entry / User / Login / Pages',
 }
 
+const STR_REGISTER = "Don't have an account yet?";
```

Comment:
This isn't interactive and should therefore be part of the element, not component